CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.6 (2016-06-03)
```

Comment:
Change this to just `## master`. I'll tag it and add a date when we finally release it.